### PR TITLE
Updated the addCapabilityToDB method from DTO to optionally create JobFamily and link it to capability.

### PR DESCRIPTION
**Pull request by PrzemekJurewicz**

### CODE REVIEW

File: src/main/java/com/kainos/ea/resources/Capability.java [hunk 1]
```diff
@@ -60,4 +69,13 @@ public String getLeadPhoto() {
...
     }
+
+    public String getCapabilityJobFamilyName() {
+        return capabilityJobFamilyName;
+    }
+
+    public void setCapabilityJobFamilyName(String capabilityJobFamilyName) {
+        this.capabilityJobFamilyName = capabilityJobFamilyName;
+    }
+
```
Comment (jkelso662) [inline src/main/java/com/kainos/ea/resources/Capability.java]: Move this white space line up to between line 19 & 20 please.

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Only one white space issue then I will approve.

**Comment by jkelso662** [comment]
Also no PR description again 😢

**Review by bbell-99 — APPROVED** [review]

**Review by jkelso662 — APPROVED** [review]
♥️

File: src/main/java/com/kainos/ea/resources/Capability.java (hunk 1 above)
Comment (PrzemekJurewicz) [reply src/main/java/com/kainos/ea/resources/Capability.java]: Done! Glad You've found it- it could have destroyed the master branch

Comment (jkelso662) [reply src/main/java/com/kainos/ea/resources/Capability.java]: Most certainly. The whole system would be at risk if I hadn't have caught this 😄

**Pull request merged**